build.gradle

```diff
@@ -55,6 +55,10 @@ test {
     systemProperty "org.gradle.android.test.gradle-installation", System.getProperty("org.gradle.android.test.gradle-installation")
     systemProperty "org.gradle.android.test.android-version", System.getProperty("org.gradle.android.test.android-version")
     systemProperty "org.gradle.android.test.scan-url", System.getProperty("org.gradle.android.test.scan-url")
+    systemProperty "org.gradle.smoketests.mirror.init.script", System.getProperty("org.gradle.smoketests.mirror.init.script")
+    if(System.getProperty('org.gradle.internal.plugins.portal.url.override')) {
```

Comment:
Space after `if` pls.

Reply:
Also, please extract the value of the property to a local variable instead of querying it twice.